tests/integration.rs

```diff
@@ -4073,8 +4073,8 @@ mod common_parallel {
     fn test_serial_socket_interaction() {
         let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
         let guest = Guest::new(Box::new(focal));
-        let serial_socket = guest.tmp_dir.as_path().join("/tmp/serial.socket");
-        let serial_socket_pty = guest.tmp_dir.as_path().join("/tmp/serial.pty");
```

Comment:
This might fix some of the flake issues we were seeing with the `test_serial_socket_interaction` test as using absolute names replaces the path so two tests would be using the same file....